bundle/nginx/nginx.conf.template

```diff
@@ -0,0 +1,95 @@
+events {
```

Comment:
please configure default redirect from http to https

Reply:
I don't add it because than i need add the port to cfg.